ui/apps/platform/eslint-plugins/pluginGeneric.js

```diff
@@ -0,0 +1,139 @@
+/* globals module */
+
+const rules = {
+    // ESLint naming convention for positive rules:
+    // If your rule is enforcing the inclusion of something, use a short name without a special prefix.
+
+    'ExternalLink-a': {
+        // Require ExternalLink and a elements for consistent presentation.
+        // For example, ExternalLinkAltIcon follows a element.
+        meta: {
+            type: 'problem',
+            docs: {
+                description: 'Require ExternalLink and a elements for consistent presentation',
```

Comment:
nit: This was also a bit hard to understand. Is the following correct?

```Require consistent presentation of ExternalLink and <a> elements, with <a> as a child of ExternalLink.```

Reply:
Indeed, thank you for reader viewpoint.